survey_app/src/androidTest/java/org/opendatakit/survey/BasicTest.java

```diff
@@ -0,0 +1,20 @@
+package org.opendatakit.survey;
+
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import static org.junit.Assert.assertTrue;
+
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class BasicTest {
+
+    @Test
+    public void testFailureUsedToVerifyBuildsCatchWhenSet() {
+        //assertTrue(false);
+        assertTrue(true);
```

Comment:
What purpose is this serving? The name implies it was meant to fail, but now it seems like its meant to pass as a no-op? Should we rename? Or remove?